powermon/commands/result.py

```diff
@@ -78,7 +89,7 @@ def decode_response(self, raw_response) -> list[Reading]:
 
         # Process response based on result type
         match self.result_type:
-            case ResultType.SINGLE:
+            case ResultType.SINGLE | ResultType.ERROR:
```

Comment:
probably needs own case for .ERROR

Reply:
I'll split it out but it will be the same for now.